maestro-orchestra-models/src/main/java/maestro/orchestra/util/InputTextRandom.kt

```diff
@@ -0,0 +1,56 @@
+package maestro.orchestra.util
+
+object InputTextRandom {
+    private const val CHARSET_TEXT = "ABCDEFGHIJKLMNOPQRSTUVWXTZabcdefghiklmnopqrstuvwxyz0123456789"
+    private const val CHARSET_NUMBER = "0123456789"
+    private const val CHARSET_NUMBER_WITHOUT_ZERO = "123456789"
+    private val LIST_POPULAR_LAST_NAME = arrayOf("Smith", "Johnson", "Williams", "Brown", "Jones", "Garcia", "Miller", "Davis", "Rodriguez", "Martinez")
+    private val LIST_POPULAR_FIST_NAME = arrayOf(
+        "Liam",
+        "Olivia",
+        "Noah",
+        "Emma",
+        "Oliver",
+        "Charlotte",
+        "Elijah",
+        "Amelia",
+        "James",
+        "Ava",
+        "William",
+        "Sophia",
+        "Benjamin",
+        "Isabella",
+        "Lucas",
+        "Mia",
+        "Henry",
+        "Evelyn",
+        "Theodore",
+        "Harper"
+    )
+    private val LIST_POPULAR_EMAIL_DOMAIN = arrayOf("gmail.com", "yahoo.com", "hotmail.com", "aol.com", "msn.com", "outlook.com")
+
+    fun randomPersonName() = String.format(
+        "%s %s",
+        LIST_POPULAR_FIST_NAME.random(), LIST_POPULAR_LAST_NAME.random()
+    )
+
+    fun randomEmail() = String.format(
+        "%s_%s@%s",
```

Comment:
Albeit there are about 200 possible combinations, at scale there would still be quite a real probability of duplicates. How about adding a alphanumerical suffix to increase entropy? I.e. `Emma_Williams_x89a@gmail.com`